test/test.go

```diff
@@ -157,3 +159,45 @@ func DescribeTestsFor(t TestCase) bool {
 		}()
 	})
 }
+
+func PrepareServiceInstance(ctx *common.TestContext, auth *common.SMExpect, platformID, planID string, OSBContext string) (string, *types.ServiceInstance) {
```

Comment:
this function does not fit to the scope of this file

Reply:
It doesn't really, I agree. Do you have any better suggestions?